salt/cloud/clouds/vmware.py

```diff
@@ -59,7 +59,7 @@
 # Import python libs
 from __future__ import absolute_import
 from random import randint
```

Comment:
Can you change this line to `from random import randint as _randint` and then change any necessary references? This will clean up the test error on Ubuntu 14.